### PR TITLE
Added and changed few things in Book.java and Inventory.java

### PR DESCRIPTION
**Pull request by tanishagarg2**
- Modified Book.java
- completed the incomplete code for Inventory.java

### CODE REVIEW

File: src/main/java/com/kyhorne/bookstore/Inventory.java [hunk 1]
```diff
@@ -9,41 +9,45 @@
...
      * @return
      */
-    //TODO: Actually write out the proper code
     public Book retrieveBook(Book bookRequested){
-        return bookRequested;
+        for(Book b: bookInventory){
+            if(b.equals(bookRequested))
+                return b;
+        }
+        return null;
```
Comment (kyhorne) [inline src/main/java/com/kyhorne/bookstore/Inventory.java]: Change to: `bookInventory.get(bookRequested);`

File: src/main/java/com/kyhorne/bookstore/Inventory.java [hunk 2]
```diff
@@ -9,41 +9,45 @@
...
      * @param bookRequested
      * @return
      */
-    //TODO: Actually write out the proper code
     public boolean containsBook(Book bookRequested){
+        for(Book b: bookInventory) {
+            if (b.equals(bookRequested))
+                return true;
+        }
         return false;
```
Comment (kyhorne) [inline src/main/java/com/kyhorne/bookstore/Inventory.java]: Change to: `return bookInventory.contains(bookRequested);`

File: src/main/java/com/kyhorne/bookstore/Inventory.java [hunk 3]
```diff
@@ -9,41 +9,45 @@
  */
 public class Inventory {
 
-    //
     private ArrayList<Book> bookInventory;
```
Comment (kyhorne) [inline src/main/java/com/kyhorne/bookstore/Inventory.java]: Change to: `private List<Book> bookInventory;`

File: src/main/java/com/kyhorne/bookstore/Inventory.java [hunk 4]
```diff
@@ -5,46 +5,47 @@
...
      */
-    //TODO: Actually write out the proper code
     public Book retrieveBook(Book bookRequested){
-        return bookRequested;
+        for(Book b: bookInventory){
+            if(b.equals(bookRequested))
+                return b;
+        }
+        return null;
     }
```
Comment (kyhorne) [inline src/main/java/com/kyhorne/bookstore/Inventory.java]: I don't know why this method is necessary...

Comment (tanishagarg2) [reply src/main/java/com/kyhorne/bookstore/Inventory.java]: Just to get the book from the inventory if necessary. I can get rid of it if it seems completely useless

**Pull request closed without merging**